### ISSUE TITLE
Speed up and parallelise building point extraction

### ISSUE THREAD

**Issue by dwastberg** [dtcc-platform/dtcc-builder#34]
Tile the building footprints and pointcloud points in a way that each tile can be run independently and possibly in parallel.

**Comment by dwastberg** [comment]
Developing in tile_city_processing branch

**Comment by dwastberg** [comment]
merged into develop. Can still be faster, but good enough for now
